js/flightlog.js

```diff
@@ -578,11 +578,20 @@ function FlightLog(logData) {
 
                     // Add the Feedforward PID sum (P+I+D+F)
                     for (var axis = 0; axis < 3; axis++) {
-                        destFrame[fieldIndex++] =
+                        let pidSum =
                             (axisPID[axis][0] !== undefined ? srcFrame[axisPID[axis][0]] : 0) +
                             (axisPID[axis][1] !== undefined ? srcFrame[axisPID[axis][1]] : 0) +
                             (axisPID[axis][2] !== undefined ? srcFrame[axisPID[axis][2]] : 0) +
                             (axisPID[axis][3] !== undefined ? srcFrame[axisPID[axis][3]] : 0);
+
+                        // Limit the PID sum by the limits defined in the header
+                        let pidLimit = axis < AXIS.YAW ? sysConfig.pidSumLimit : sysConfig.pidSumLimitYaw;
+                        if (pidLimit) {
```

Comment:
I assume what you want to test here is `pidLimit !== 0`? While simplified 0 checking is fine for C, using it in JS is dangerous, due to the very wide definition of truthty and falsey.

Reply:
I tested it with null, zero and values different than zero. But I can change it tomorrow to something more explicit.

Reply:
Pushed a change .